lib/index.js

```diff
@@ -19,6 +19,7 @@ module.exports = function nuxtAxios (moduleOptions) {
     proxyHeaders: true,
     proxyHeadersIgnore: ['accept', 'host'],
     debug: false,
+    disableErrorHandler: false,
```

Comment:
What's your idea changing it to `errorHandler: true`

Reply:
Take a look at #33. Maybe you want to handle your errors together with some other logic in the responseInterceptor. With the current design the `responseInterceptor` does not get called if an error happens.

Reply:
It's just a suggestion. I can also remove this.
For me it works fine, when I'm overriding the default error handler with the custom error handler I defined.

Reply:
He suggested that option name can be change from disableErrorHandler to errorHandler

Reply:
Changes in this PR looks good to me. Just think this option name may be little confusing for users ... Maybe either : 
- A new option like `disableDefaultErrorHandler`
- Handle explicit `false` value in else condition with: `else if (options.disableErrorHandler === false)`

BTW thanks for working on this...

Reply:
Yeah you are right. I will change this on occasion.

No problem, I‘m thankful for the Package :)